tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -202,6 +202,25 @@ def test_bf_cross_section(packet_params, expected, packet, model):
     assert_almost_equal(obtained, expected)
 
 
+# TODO: fix underlying method and update expected values in testcases.
+# For loop is not being executed in original method, and hence bf_helper
+# always remains zero. Reason for for loop not executed:
+#         "current_continuum_id = no_of_continuum edges"
+@pytest.mark.parametrize(
+    ['packet_params', 'expected'],
+    [({'nu': 0.1, 'mu': 0.3, 'r': 7.5e14}, 0.0),
+     ({'nu': 0.2, 'mu': -.3, 'r': 7.7e14}, 0.0)]
+)
```

Comment:
@wkerzendorf @yeganer There might be a problem in `calculate_chi_bf`s implementation, or I wonder `chi_bf` is zero for these test cases. I found a _FIXME_ note in the underlying method, and I suspect that it is indeed not thread-safe.